enterprise/admin/__init__.py

```diff
@@ -1004,13 +1004,26 @@ class EnterpriseCustomerInviteKeyAdmin(admin.ModelAdmin):
         'enterprise_customer',
         'usage_limit',
         'expiration_date',
+        'is_active',
     )
```

Comment:
Unrelated: while we're in here, should we add the ability to search for invite keys via the EnterpriseCustomer.{slug, name, and/or UUID}? Imagine that will be helpful in the not too distant future for ourselves and ECS.